solutions/usaco-990.mdx

```diff
@@ -212,4 +212,41 @@ public class berries {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+with open('berries.in') as read:
+	n, k = map(int, read.readline().split())
+	berries = [*map(int, read.readline().split())]
+
+maximum, ans = max(berries), 0
+
+for i in range(1, maximum + 1):
+	mod, full, tmp = i, 0, 0
+
+	for j in range(n):
+		full += berries[j] // mod
+	if full < k / 2:
+		break
```

Comment:
add comments explaining what you're doing
NOTE: do _not_ add comments explaining the obvious, like "increment the answer by this amount" or "print the answer"
add comments explaining like "while we can still [x], do [x]" from a high-level perspective

Reply:
okay